server/routes/tracker.js

```diff
@@ -5,7 +5,7 @@ const router = express.Router();
 
 router.get('/example', trackerController.exampleGet);
 
-router.get('/getList', trackerController.getList);
+router.get('/getList/:renderAmount/:user', trackerController.getList);
```

Comment:
I think we can remove the example route & controller function now! Also consider a restful implementation in the future.

Reply:
WIll keep in mind and update it to be a more restful route in the near future.